src/main/flight/pid.c

```diff
@@ -1331,44 +1356,38 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, timeUs_t currentTim
             }
 #endif
 
-#if defined(USE_D_CUT)
-            if (dtermCutPercent){
-                dtermCutFactor = dtermCutRangeApplyFn((filter_t *) &dtermCutRange[axis], delta);
-                dtermCutFactor = fabsf(dtermCutFactor) * dtermCutGain;
-                dtermCutFactor = dtermCutLowpassApplyFn((filter_t *) &dtermCutLowpass[axis], dtermCutFactor);
-                dtermCutFactor = MIN(dtermCutFactor, 1.0f);
-                dtermCutFactor = dtermCutPercentInv + (dtermCutFactor * dtermCutPercent);
-            }
-            if (axis == FD_ROLL) {
-                DEBUG_SET(DEBUG_D_CUT, 2, lrintf(pidCoefficient[axis].Kd * dtermCutFactor * 10.0f / DTERM_SCALE));
-            } else if (axis == FD_PITCH) {
-                DEBUG_SET(DEBUG_D_CUT, 3, lrintf(pidCoefficient[axis].Kd * dtermCutFactor * 10.0f / DTERM_SCALE));
+            float dMinFactor = 1.0f;
+#if defined(USE_D_MIN)
+            if (dMinPercent[axis]) {
+                float dMinFactor = 1.0f;
```

Comment:
there's probably no harm from this, but now that it's also done at L1359, we don't need it here.

Reply:
Actually there is harm in this. The second declaration will only be in the scope of the `if (dMinPercent[axis]) {` block and calculated `dMinFactor` will lose scope below it and the D calculation will always use 1.0.

Reply:
Thanks eTracer for looking so closely into this, and fixing so many potential issues - I really do appreciate your advice and the time you take on these things.